install_files/ansible-base/usr.sbin.apache2

```diff
@@ -206,6 +206,7 @@
   /var/www/securedrop/static/i/tipbox/tipbox-hed-submit3.png r,
   /var/www/securedrop/static/i/tipbox/tipbox-logo.png r,
   /var/www/securedrop/static/js/journalist.js r,
+  /var/www/securedrop/static/js/libs/jquery-2.1.4.js r,
```

Comment:
You're right that's how AppArmor works, was just curious if making this file readable by users of SecureDrop web apps is strictly necessary to be compliant with AGPL if the uncompressed js were to be distributed only in the repo (e.g. we provide `.py` code only in the repo). Either way, this is OK, will approve for merge 🙃